code/modules/games/cards.dm

```diff
@@ -47,12 +51,57 @@
 	var/card_attack_verb
 	/// Inherited card resistance
 	var/card_resistance_flags = FLAMMABLE
-
-/obj/item/deck/Initialize(mapload)
+	/// ID used to track the decks and cardhands that can be combined into one another.
+	var/main_deck_id = -1
+	/// The name of the last player to interact with this deck.
+	var/last_player_name
+	/// The action that the last player made. Should be in the form of "played a card", "drew a card."
+	var/last_player_action
+
+/obj/item/deck/Initialize(mapload, parent_deck_id = -1)
 	. = ..()
 	build_decks()
-	update_icon(UPDATE_ICON_STATE)
+	update_icon(UPDATE_ICON_STATE|UPDATE_OVERLAYS)
+	if(parent_deck_id == -1)
+		// we're our own deck
+		main_deck_id = rand(1, 99999)
+	else
+		main_deck_id = parent_deck_id
 
+	AddComponent(/datum/component/proximity_monitor/table)
+	RegisterSignal(src, COMSIG_ATOM_RANGED_ATTACKED, PROC_REF(on_ranged_attack))
+
+
+/obj/item/deck/examine(mob/user)
+	. = ..()
+	. += "<span class='notice'>It contains <b>[length(cards) ? length(cards) : "no"]</b> card\s.</span>"
+	if(last_player_name && last_player_action)
+		. += "<span class='notice'>Most recent action: [last_player_name] [last_player_action].</span>"
+	if(in_play_range(user) && !Adjacent(user))
+		. += "<span class='boldnotice'>You're in range of this card-hand, and can use it at a distance!</span>"
+	else
+		. += "<span class='notice'>You're too far away from this deck to play from it.</span>"
+	. += ""
+	. += "<span class='notice'>Drag [src] to yourself to pick it up.</span>"
+	. += "<span class='notice'>Examine this again to see some shortcuts for interacting with it.</span>"
+
+/obj/item/deck/examine_more(mob/user)
+	. = ..()
+	. += "<span class='notice'><b>Click</b> to draw a card.</span>"
+	. += "<span class='notice'><b>Alt-Click</b> to place a card.</span>"
+	. += "<span class='notice'><b>Ctrl-Shift-Click</b> to split [src].</span>"
+	. += "<span class='notice'>If you draw or return cards with <span class='danger'>harm</span> intent, your plays will be public!</span>"
+	. += "<span class='notice'>With cards in your active hand...</span>"
+	. += "\t<span class='notice'><b>Ctrl-Click</b> with cards to place them at the bottom of the deck.</span>"
+	. += ""
+	. += "You also notice a little number on the corner of [src]: it's tagged [main_deck_id]."
```

Comment:
I want this one to be subtle, since it's more informational but not really something that needs to be highlighted. Maybe I can set the font size smaller or use <small> or something

Reply:
That or there might be another class you could apply to it instead. We do have a lot of them, not sure if there was something that would work better in this situation?